.github/workflows/release.yml

```diff
@@ -275,19 +275,19 @@ jobs:
 
```

Comment:
The code change looks mostly correct, but there seems to be a small mistake in the assignment of the `RELEASE_NOTES` variable.